.github/workflows/ruby-ci.yml

```diff
@@ -0,0 +1,17 @@
+name: Ruby-CI
+on: [pull_request]
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1.2
+          bundler-cache: true
+      - name: Run tests
+        run: bundle exec rake
```

Comment:
the command should be `bundle exec rake test` :)

Reply:
Oh yes, Thank you :)